react-search/my-app/src/ListItems.tsx

```diff
@@ -0,0 +1,7 @@
+type Props = {
+  mappedQuotes: JSX.Element[];
```

Comment:
It would be best to pass the data rather than the JSX elements. The vast majority of times your props will be data.